s3am/Makefile

```diff
@@ -0,0 +1,25 @@
+# Definitions
+build_tool = runtime-container.DONE
+name = quay.io/ucsc_cgl/s3am
+tag = 1.0a1
```

Comment:
right, in conversation with @hannes-ucsc he suggested we might still want the commit of cgl-docker-lib in the tag.